x-pack/plugins/apm/public/components/shared/StickyProperties/index.tsx

```diff
@@ -59,11 +60,17 @@ const PropertyValueTruncated = styled.span`
 `;
 
 function TimestampValue({ timestamp }: { timestamp: Date }) {
+  const notAvailableLabel = i18n.translate(
+    'xpack.apm.timestampValue.notAvailableLabel',
+    {
+      defaultMessage: 'N/A'
+    }
+  );
```

Comment:
I wonder if we should just have a single "not available" label in some helper lib, instead of duplicating it everywhere.  What do you think @maryia-lapata ?